index.js

```diff
@@ -2,22 +2,35 @@
 
 var axios = require('axios')
 var axiosDebug = require('debug')('axios')
+var querystring = require('querystring')
```

Comment:
I have a concern that if the querystring module is good here due to axios-debug-log is designed to be an isomorphic module. After querystring module is referenced, user could just use it in node.js or module bundler including polyfills of node.js like webpack.

Reply:
Actually I wonder if we can borrow this part from axios itself to ensure the log and the actual request are the same, even axios have some bug.

Reply:
Ah right! I've pushed a new commit using axios's `buildURL` helper. In the [latest version which is still in beta](https://github.com/axios/axios/releases/tag/v0.19.0-beta.1), axios will expose a [`getUri`](https://github.com/axios/axios/pull/1712) method, but I didn't use it to make sure that the debugger still works with older versions.

This should also resolve your other comment (which I didn't think about, thanks :) ).